orders/serializers.py

```diff
@@ -11,6 +11,16 @@ class Meta:
         fields = "__all__"
 
 
+class CreateShortOrderSerializer(serializers.ModelSerializer):
```

Comment:
lower case every thing 
only constants and classes get to have upper case letters